test/perf/micro/perf.f90

```diff
@@ -190,6 +190,15 @@ integer function parse_int(s, base) result(n)
 
 end function
 
+subroutine printfd(n)
+integer, intent(in) :: n
+integer :: i , unit
+open(unit=1, file="foo")
```

Comment:
Don't use units directly like that. Rather use `newunit`:
```fortran
integer :: u
open(newunit=u, file="foo", status="replace")
```
Also, very important: you must put the `status` in there. Per the Fortran standard, if the `status` specifier is omitted, the default value is `UNKNOWN`. The `UNKNOWN` status is processor dependent.

See here for more details: http://www.fortran90.org/src/best-practices.html#file-input-output